test/parallel/test-cluster-worker-init.js

```diff
@@ -3,30 +3,25 @@
 // verifies that, when a child process is forked, the cluster.worker
 // object can receive messages as expected
 
-require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var msg = 'foo';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const msg = 'foo';
 
 if (cluster.isMaster) {
-  var worker = cluster.fork();
-  var timer = setTimeout(function() {
-    assert(false, 'message not received');
-  }, 5000);
+  const worker = cluster.fork();
 
-  timer.unref();
-
-  worker.on('message', function(message) {
+  worker.on('message', common.mustCall((message) => {
```

Comment:
I'd prefer to keep the other `common.mustCall()`'s because they are actually mandatory for passing the test too. At least it gives more granularity to error msg if test fails if we have them.